api/v1/env.go

```diff
@@ -302,6 +301,14 @@ func (s *Store) GetEnv() []corev1.EnvVar {
 			Name:  "SQL_SET_DEFAULT_SESSION_VARIABLES",
 			Value: "0",
 		},
+		{
+			Name:  "INSTALL_LOCALE",
+			Value: s.Spec.ShopConfiguration.Locale,
+		},
+		{
+			Name:  "INSTALL_CURRENCY",
+			Value: s.Spec.ShopConfiguration.Currency,
```

Comment:
I guess the key for the helm chart is `store.currency`?

Reply:
No for the crd it is: `store.shopConfiguration.currency`

Reply:
You can see that in the `store.go` file which is the crd in the end. The main entry point is `StoreSpec` from there you can follow the types for example `ShopConfiguration Configuration `json:"shopConfiguration,omitempty"`` which is then a new type and so further.

Reply:
Ok thanks.
Then it might be worth adding this into `values.yaml` in the Helm chart (it there is not already a PR that does it).